src/movies.js

```diff
@@ -1,16 +1,149 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(arr) {
+    let newArr = arr.map(function(elem){
```

Comment:
Try avoiding names like newArr when you can be little more descriptive when naming your variables, it is a small thing but helps greatly in the end! something like directorsArr could be better here :)